### DISCUSSION TITLE
Issue Forms: Add ability for dropdown/checkbox choices to add additional labels on issue creation

### DISCUSSION THREAD

**Discussion by just-joshing** [community/community#15807] [Projects and Issues]
I want to be able to specify labels associated with dropdown/checkbox options on the form that should get added when the issue is created.

### More context:
My team tracks issues by tagging them with labels for the scope of the issue. For example, one issue might have the labels [`bug`, `area:ProjectA`, `scope:Organization`] whereas another might have [`bug`, `area:ProjectB`, `scope:Repository`].

I have some multiselect dropdowns in my issue form for users to specify the area and scope of a bug report. But beyond the basic `bug` label that the overall template/form applies, someone on my team has to look at the chosen options in the created issue and manually apply the labels for the options chosen in the form.

I would love it if I could add a label attribute to an option in the template/form so that when that option is chosen by a user making an issue, the resulting issue automatically applies the label.

**Comment by assignUser** [comment]
This would be a great addition for open source projects too. We are in the process of migrating from JIRA to GitHub Issues with the apache/arrow project and we have to do this via a workflow that parses the issue body as only a select number of people have enough permissions to add labels after creation and having 15+ templates wouldn't really be great UX.

**Reply by stephhazlitt** [reply]
+1 🙏

**Reply by pitrou** [reply]
And of course parsing the issue body to reconstruct the information isn't exactly robust while the information was ipso facto available at form submission time...

**Comment by jrhemstad** [comment]
This would be a fantastic feature and I was disappointed when I learned this wasn't already available. 

As a stop gap, I found that there is an existing action workflow that will do this: https://github.com/marketplace/actions/advanced-issue-labeler

That said, it would be far better to have built-in support and not have to rely on 3rd party, unverified action workflows for such a simple and intuitive feature.

**Comment by meeas** [comment]
Yes, please!  This would greatly help my projects as well!
